From d98fa2ee2ec81018ef11194f3b7edc4db9ffbc6a Mon Sep 17 00:00:00 2001
From: Katherine Lu <katherine.lu@freescale.com>
Date: Tue, 26 Aug 2008 16:01:21 +0800
Subject: [PATCH] ENGR00088057 MX35: Disable UART2 DMA mode

Disabled the UART2 DMA mode for it caused data lost when hardware
flow control disabled. Bluetooth now works in no hw flow mode.

Signed-off-by: Katherine Lu <katherine.lu@freescale.com>
---
 arch/arm/mach-mx35/serial.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx35/serial.h b/arch/arm/mach-mx35/serial.h
index 434dd8d..232ecf7 100644
--- a/arch/arm/mach-mx35/serial.h
+++ b/arch/arm/mach-mx35/serial.h
@@ -62,7 +62,7 @@
 /* UART 2 configuration */
 #define UART2_HW_FLOW           1
 #define UART2_UCR4_CTSTL        16
-#define UART2_DMA_ENABLE        1
+#define UART2_DMA_ENABLE        0
 #define UART2_DMA_RXBUFSIZE     1024
 #define UART2_UFCR_RXTL         16
 #define UART2_UFCR_TXTL         16
-- 
1.5.4.4

